mmocr/apis/inference.py

```diff
@@ -6,23 +6,40 @@
 from mmdet.datasets.pipelines import Compose
 
 
-def disable_text_recog_aug_test(cfg):
+def disable_text_recog_aug_test(cfg, set_types=None):
     """Remove aug_test from test pipeline of text recognition.
     Args:
         cfg (mmcv.Config): Input config.
+        set_types (list[str]): Type of dataset source. Should be
```

Comment:
What does "type of dataset" mean?

Reply:
`val` or `test`. For validation, it shpuld be `['val']`. For test, it should be `['test']`